doc/configcommands.dsv

```diff
@@ -125,7 +125,7 @@ ttrss-password||<password>||""||Configures the password for use with Tiny Tiny R
 ttrss-passwordfile||<path>||""||A more secure alternative to the above, by storing your password elsewhere in your system.||ttrss-passwordfile "~/.newsboat/ttrss-pw.txt"
 ttrss-passwordeval||<command>||""||Another secure alternative, is providing your password from an external command that is evaluated during login. This can be used to read your password from a gpg encrypted file or your system keyring.||ttrss-passwordeval "command some-parameter"
 ttrss-url||<url>||""||Configures the URL where the Tiny Tiny RSS installation you want to use resides.||ttrss-url "http://example.com/ttrss/"
-unbind-key||<key> [<dialog>]||n/a||Unbind key <key>. This means that no operation is called when <key> is pressed. Optionally, you can specify a dialog (for a list of available dialogs, see `bind-key` above). If you specify one, the key binding will only be unbound for the specified dialog.||unbind-key R
+unbind-key||<key> [<dialog>]||n/a||Unbind key <key>. This means that no operation is called when <key> is pressed. Optionally, you can specify a dialog (for a list of available dialogs, see `bind-key` above). If you specify one, the key binding will only be unbound for the specified dialog. If you provide "-a" as <key>, all keys (default ones and keys specified to this point) will be unbinded.||unbind-key R
```

Comment:
Oh, I didn't remember about contexts when I wrote my comments in #119. I think they should be taken into account, i.e.:
* `unbind-key -a` unbinds everything;
* `unbind-key -a articlelist` should unbind all the articlelist-specific bindings, but keep everything else.

Can you please add that?

I'd replace "all keys (default ones and keys specified to this point)" with "all currently bound keys", and move that whole sentence before "Optionally". That way, sentences describe every part of the command in order: first "unbind-key", then "\<key>", and finally "[dialog]".

Reply:
Should I test that key is not set for every context in the tests? (That would add a lot of REQUIREs)

Reply:
`KeyMap::getkey(key, "all")` goes through all contexts in search for the `key`, so I think you'll be fine with `REQUIRE(keymap.getkey(key, "all") == "<none>")`.

(It'd be awesome of you to add a test to KeyMap that verifies that behaviour, so we won't accidentally break it in the future :)